plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -308,6 +308,14 @@ public Object start(IApplicationContext context) {
         }
     }
 
+    private void overrideTimezone() {
+        DBPPreferenceStore preferenceStore = DBWorkbench.getPlatform().getPreferenceStore();
+        final String timezone = preferenceStore.getString(DBeaverPreferences.CLIENT_TIMEZONE);
+        if (timezone != null && !timezone.equals("DEFAULT")) {
+            TimeZone.setDefault(TimeZone.getTimeZone(timezone));
+        }
+    }
+
```

Comment:
I'm starting to think that this is what the introduced `TimezoneRegistry` should be used for.